### ISSUE TITLE
Nested resources are returning all resources

### ISSUE THREAD

**Issue by ronhook** [typicode/jsonplaceholder#91]
When calling a GET for https://jsonplaceholder.typicode.com/posts/1/comments all comments for every post is returned. Expected behaviour is that only comments with postId of 1 should be returned. This is occurring in all of the nested resource examples.

**Comment by santoshyadav198613** [comment]
Hi I will take care of this,

**Comment by AnnieMfsi** [comment]
the issue is still persistent. Any update on this?

**Comment by Riyenz** [comment]
@typicode I tried to remove the `_isFake` option in `app.js` and the nested resources works properly. I dunno what isFake options does but I think this needs to be fix in json-server repo?

```
/** FROM */
const router = jsonServer.router(clone(data), { _isFake: true })
/** TO */
const router = jsonServer.router(clone(data))

```

**Comment by kwdowik** [comment]
@Riyenz  You are right. I checked in json-server code and found out that router get 2 arguments
`module.exports = (db, opts = { foreignKeySuffix: 'Id', _isFake: false }) => {`

and then in code 

`// Handle /:parent/:parentId/:resource
  router.use(nested(opts))`

Also found a test which covers a case that you describe above

` describe('GET /:parent/:parentId/:resource', () => {
    test('should respond with json and corresponding nested resources', () =>
      request(server)
        .get('/posts/1/comments')
        .expect('Content-Type', /json/)
        .expect([db.comments[0], db.comments[1]])
        .expect(200))
  })`

and this server is pass as an argument to the request and it is created in beforeEach method without set _isFake property (in default is set to false) and test, of course, has passed, when I have set  _isFake to true as an argument to jsonServer.router method test has failed and returned all resources.

So this investigation proves that is not jsonplachelder fault. This _isFake property was introduced 7 months ago in json-server and also 7 months ago _isFake property has been added to jsonplaceholder in my opinion by mistake but I am not sure.

Cheers

**Comment by AnnieMfsi** [comment]
@typicode

**Comment by yashshah1** [comment]
Can I take this up?

**Comment by yashshah1** [comment]
https://github.com/typicode/json-server/blob/8dd0b456ab313898f094c1e113f647425b2898a4/src/server/router/nested.js#L9

I think this is by design in the json-server repository, and if a fix is needed, an issue should be raised there.

**Comment by evankrocker** [comment]
How do you add the rest of the json file?  Tried to add strings but xcode keeps crashing
